torch/_refs/__init__.py

```diff
@@ -2206,7 +2206,11 @@ def _make_copy_from_view(fn):
     """
     Given a view function (e.g. torch.diagonal) generates its copy variant (e.g. torch.diagonal_copy)
     """
-    fn = out_wrapper()(getattr(aten, fn.__name__))
+    aten_fn = getattr(aten, fn.__name__)
+    if not hasattr(aten_fn, "__annotations__"):
+        aten_fn.__annotations__ = dict(fn.__annotations__)
```

Comment:
Let's not modify global state here pls.

Reply:
I didn't do that without thinking, but I thought wrong. My idea was that aten functions should have `__annotations__` but it's a silly idea to add just a few.

I added a new parameter to `out_wrapper`, which does it.